### PR TITLE
Updated storage versions post January release

### PR DESCRIPTION
**Pull request by gapra-msft**

### CODE REVIEW

File: sdk/core/azure-core-http-netty/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Azure Core Netty HTTP client is a plugin for the azure-core HTTP client API.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-http-netty</artifactId>
-    <version>1.2.0</version>
```
Comment (gapra-msft) [inline sdk/core/azure-core-http-netty/README.md]: Do I undo these non storage ones?

Comment (alzimmermsft) [reply sdk/core/azure-core-http-netty/README.md]: Yes, the non-Storage versions in READMEs should be reverted. Run the update command with `--sr` to skip updating README files, or just manually revert the OkHttp and Netty changes.

**Comment by alzimmermsft** [comment]
The changes to the version file look good, I think you'll want to re-run the update script without changing the READMEs as those had the latest released versions listed which is what we want.

File: sdk/core/azure-core-http-netty/README.md (hunk 1 above)
Comment (gapra-msft) [reply sdk/core/azure-core-http-netty/README.md]: Done!

**Comment by gapra-msft** [comment]
> The changes to the version file look good, I think you'll want to re-run the update script without changing the READMEs as those had the latest released versions listed which is what we want.

@alzimmermsft Can you re-take a look? I undid the README changes

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
-    <version>1.0.0-beta.2</version>
+    <version>1.0.0-beta.3</version>
```
Comment (alzimmermsft) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md]: This needs to be reverted.

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/CreateConfigurationSettingTracingSample.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
-    <version>1.0.0-beta.2</version>
+    <version>1.0.0-beta.3</version>
```
Comment (alzimmermsft) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/CreateConfigurationSettingTracingSample.md]: This needs to be reverted.

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
-    <version>1.0.0-beta.2</version>
+    <version>1.0.0-beta.3</version>
```
Comment (alzimmermsft) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md]: This needs to be reverted.

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/PublishEventsTracingSample.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** for implementation and **
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
-    <version>1.0.0-beta.2</version>
+    <version>1.0.0-beta.3</version>
```
Comment (alzimmermsft) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/PublishEventsTracingSample.md]: This needs to be reverted.

**Pull request merged**